packages/web/components/complex/add-conc-liquidity.tsx

```diff
@@ -623,6 +623,27 @@ const StrategySelectorGroup: FunctionComponent<
   } & CustomClasses
 > = observer((props) => {
   const t = useTranslation();
+  const { volatilityType } = props.addLiquidityConfig;
+
+  let descriptionText = t(
+    "addConcentratedLiquidity.volatilityCustomDescription"
+  );
+
+  if (volatilityType === "passive") {
+    descriptionText = t(
+      "addConcentratedLiquidity.volatilityPassiveDescription"
+    );
+  } else if (volatilityType === "aggressive") {
+    descriptionText = t(
+      "addConcentratedLiquidity.volatilityAggressiveDescription"
+    );
+  } else if (volatilityType === "moderate") {
+    descriptionText = t(
+      "addConcentratedLiquidity.volatilityModerateDescription"
+    );
+  } else if (volatilityType === "custom") {
+    descriptionText = t("addConcentratedLiquidity.volatilityCustomDescription");
+  }
```

Comment:
Given how it's used here, what if we change the `volatilityType` to be "passive" | "aggressive" | "moderate" | null

Then we don't do the last check for "custom" and just leave it as the initialized value.